src/System.ComponentModel.TypeConverter/tests/Design/MenuCommandTests.cs

```diff
@@ -98,6 +98,12 @@ public static IEnumerable<object[]> ToString_TestData()
                 },
                 "00000000-0000-0000-0000-000000000000 : 0 : Supported|Enabled|Visible|Checked"
             };
+
+            // .NET Framework throws NullReferenceException.
+            if (!PlatformDetection.IsFullFramework)
+            {
+                yield return new object[] { new MenuCommand(new EventHandler(EventHandler), null), " : Supported|Enabled|Visible" };
+            }
```

Comment:
Do these tests run on netfx?
cc: @ViktorHofer

Reply:
If the test project doesn't have a netfx configuration then not - in this case not.

Reply:
So should I remove all the `PlatformDetection` stuff?

Reply:
All .NetFramework related code